ui/src/dms/MessagesList.tsx

```diff
@@ -46,7 +46,8 @@ export default function MessagesList({ filter }: MessagesListProps) {
 
       return true; // is all
     })
-    .sort(sortOptions[RECENT]);
+    .sort(sortOptions[RECENT])
+    .reverse();
```

Comment:
this mutates in place, that's fine right, because there's only one reference to it?

Reply:
this is the only place we're sorting DMs at the moment, and this is the only place we're doing this filtering logic w/ the list I'm 99% sure

Reply:
DMs/Chat Channels/ETC